packages/x-date-pickers/src/AdapterMoment/index.ts

```diff
@@ -29,7 +27,6 @@ const formatTokenMap: MuiFormatTokenMap = {
   Y: 'year',
   YY: 'year',
   YYYY: 'year',
-  YYYYYY: 'year',
```

Comment:
This format looks like `-001970 -001971 ... +001907 +001971`

It's not a valid format for us, we don't handle `-` and `+` signes